README.md

```diff
@@ -0,0 +1,40 @@
+Description
+
+Your portfolio is one of the most powerful tools in my software developer toolbox; it’s the easiest way of showing what I am truly capable of as a developer...
+
+
+Projects list
+
+Here is the list of projects that will guide me:
+Project 1: Setup and mobile version skeleton
+Project 2: Mobile version
+Project 3: Contact form (pair programming)
+Project 4: Desktop version
+Project 5: Application deploy
+Project 6: Evaluate accessibility (pair programming)
+Project 7: Mobile menu (pair programming)
+Project 8: Details modal (pair programming)
+Project 9: Validate contact form (pair programming)
+Project 10: Preserve data in the browser (pair programming)
+
+
+Mobile first approach
+
+My portfolio is prepared for different screen sizes. However, I built my portfolio by using "mobile first" approach. 
+
+Project requirements
+
+For this first project, I used the HTML & CSS linter.
+The 2 sections I built for this project are:
+  - the toolbar (or header):
+  - the headline section (right after the header):
+I chosed the Laura Jong's template: https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10.
+
+In order to lay out the elements on the page I used Flexbox in all 2 sections.
+I didn't use JavaScript, only html & css.
+
+I was stuck to the design as much as possible (e.g. font, colors, images, text, margins) using the templates in Figma.
+
+I implemented the button interactions (enable, hover, etc.).
+
+NB: In this project 1 I worked individually
```

Comment:
- Please, kindly use this readme [template](https://github.com/microverseinc/readme-template) and customize it to suit your needs for this current project.